memtable.cc

```diff
@@ -713,6 +713,39 @@ memtable::make_flat_reader(schema_ptr s,
     }
 }
 
+flat_mutation_reader
+memtable::make_reversing_reader(schema_ptr s, reader_permit permit,
+                      const dht::partition_range& range, const query::partition_slice& slice,
+                      const io_priority_class& pc,
+                      streamed_mutation::forwarding fwd) {
+    schema_ptr rev_schema_ptr = s->make_reversed();
+    if (!query::is_single_partition(range)) {
+        return make_empty_flat_reader(std::move(rev_schema_ptr), std::move(permit));
```

Comment:
I think this is incorrect. Previously I said that we don't need to support reversed partition scans, but we do. I think we support scanning in the forward direction for tokens and the reversed direction for clustering keys. Not 100% sure though.

Reply:
(and in any case, the correct response is on_internal_error(), not to silently return an incorrect result)